custom-recipes/amazon-transcribe/recipe.json

```diff
@@ -202,6 +202,23 @@
             "mandatory": true,
             "defaultValue": false
         },
+        {
+            "name": "use_timeout",
+            "label": "Use Timeout",
+            "type": "BOOLEAN",
+            "description": "Raise a timeout error in the error column if the job is longer than the specified timeout.",
```

Comment:
To me the description is too technical. WDYT about: "interrupt execution if transcriptoin job takes more time than a limit"

I thought that we also discussed an option where the timeout is always set (default 2 hours). In this case I'd only keep 1 input (no checkbox needed) and put a hint that empty value means no timeout.

Reply:
done in e1755ff1b2fdee478584ddfaf584383c280bb6a9